Modules/Core/Common/include/itkLaplacianOperator.hxx

```diff
@@ -66,7 +66,7 @@ LaplacianOperator<TPixel, VDimension, TAllocator>::GenerateCoefficients() -> Coe
 {
   // Here we set the radius to 1's, here the
   // operator is 3x3 for 2D, 3x3x3 for 3D.
-  constexpr auto r = SizeType::Filled(1);
+  static constexpr auto r{ SizeType::Filled(1) };
```

Comment:
Here is a case where the filled array may result in a larger binary. The explicit request for the compiler initialize data structure may increase the size of libraries.

Reply:
@blowekamp In practice I think the compiler should be able to inline it all away. Unless in some case when the address of the variable is taken. For example in this case, by having a pointer `p = &r`. When `r` is static, `p = &r` should always point to the same address during the run of the program. When `r` is non-static, `p = &r` may have a different address, each time it is evaluated for another instance of `r`.  However, I don't think it's common to take the address of a constexpr variable  🤷 

But of course, your main point should be measurable: does this change indeed result in a larger binary? Please feel free to do this experiment  😃